.changeset/young-roses-tickle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/starlight': patch
```

Comment:
I think we should probably make this a minor given it will cause a change in the visual appearance of people’s sites.

```suggestion
'@astrojs/starlight': minor
```